src/ome_zarr_models/v04/coordinate_transformations.py

```diff
@@ -69,3 +92,32 @@ class PathTranslation(Base):
 
     type: Literal["translation"]
     translation: str
+
+
+def ndim(transform: VectorScale | VectorTranslation) -> int:
+    """
+    Get the dimensionality of a scale or translation transform.
+    """
+    if hasattr(transform, "scale"):
+        return transform.ndim
+    elif hasattr(transform, "translation"):
+        return transform.ndim
+    else:
+        msg = f"Cannot infer the dimensionality of {type(transform)}"
+        raise TypeError(msg)
+
+
+def _build_transforms(
```

Comment:
A lazy cmd+f on GitHub doesn't give me any other hits for this function - is it used elsewhere? (this might just be because GitHub does crappy lazy loading of pages these days 😢 )

Reply:
it's now being used in a validator